chore: add comments for config fields and their usage (#12)
skip-mev/skip-go-fast-solver: README.md

### How to start server

1. Update [config/local/keys.json](config/local/keys.json) with the appropriate solver private keys and addresses

> 2. Update [config/local/config.yml](config/local/config.yml) with the appropriate config needed for each chain- all values needed to be set are placed between <> brackets. This is where you can configure the solver min/max order sizes, settlement thresholds, chain rpc links, etc. Reference the [shared/config/config.go](shared/config/config.go) file for explanations on all config fields usage

Reviewer: ```suggestion
2. Update [config/local/config.yml](config/local/config.yml) with the corresponding chain config. Values that need to be replaced are in`<>` brackets. Set these values to configure min/max order sizes, settlement thresholds, chain rpc links, etc. Reference the [shared/config/config.go](shared/config/config.go) file for more info about the config fields.
```
